parts/language/manual_ja.txt

```diff
@@ -13,50 +13,89 @@
 スピンの検出:
         「3コーナ」ルールを満たした時 +2点
         「インモービル (Immobile)」 ルールを満たした時 +2点
-        - 上のどちらかを満たした時, スピンが確定します。
+        - 上のどちらかを満たした時、スピンが確定します。
 
-        もし回転システムがキックテーブルから2番チェック (2nd check) を採用していなかった場合 +1点
-        - もしスピンの合計点が2点しかない場合は「ミニ」扱いになります；ミノがS, Z, J, L, Tのいずれかで, ラインクリア時にそのミノを全て消せなかった場合のみ。
+        もし回転システムがキックテーブルから2番目のチェック (2nd check) を採用していなかった場合 +1点
+        - もしスピンの合計点が2点しかない場合は「ミニ」扱いになります。ただしこれは、ミノがS,Z,J,L,Tのいずれかで、line消去の時にそのミノのブロックが全部消せなかった場合のみ。
 
-Attack system:
-        [TODO]
-Back to Back (B2B) gauge:
-        [TODO]
+攻撃システム:
+        Special clear:
+                Special clearとは、spin、techrash(もしくは一度に4段以上のline消去)、PC、HPCのどれかを行うことです。Special clearを行うとB2Bゲージが上昇します。
+        Super clear:
+                Super clearとは、B2Bゲージがある状態でSpecial clearを行うことです。
+        Special clears (spin):
+                消したline数の2倍の火力を送ります、
+                - B2Bは、spin single/doubleで1line、spin tripleで2line、spin techrashで4line、spin techrash+で8lineを火力に加算します。
+                - B2B2Bは、消したline数の半分と+1lineを火力に加算します。
+                - ミニは、火力を25%に減少させます。(倍率0.25倍)
+
+        Special clears (spin以外のtechrash/techrash+):
+                - B2Bは、火力を+1します。
+                - B2B2Bは、火力を50%上昇させ、更に+1line火力に加算します。
+        
+        Non-special clears:
+                火力-0.5段を送ります。
+
+        ren:
+                ren数は、連続してline消去した回数-1回です。PCとHPCを除くSupecial clearにren数*25%、1line消去にren数*15%(最大12*15%)、更に3ren以上で+1lineを火力に加算します。
+        
+        Hemi Perfect Clear (下にブロックが残っている状態でのPC、line消去した後にプレイヤーが置いたブロックが残ってはいけない):
```

Comment:
`If just one line cleared, the remaining blocks must not be player-placed` means for one-line clear to become a HPC, `the remaining blocks must not be player-placed`. If it is a 2-line clear or 3-line clear or ..., the remaining blocks **can** be player-placed.